### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by AbeeraTahir**
In this PR regarding the to-do list project, I have implemented the following functionalities:

- Created a to-do list design.
- Generated the to-do list dynamically with JavaScript.
- Used webpack

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    index: 1,
+    completed: 'true',
+    description: 'cook food',
+  },
+  {
+    index: 2,
+    completed: 'true',
+    description: 'complete project',
+  },
+];
```
Comment (codecaiine) [inline src/index.js]: - [ ] Good work so far, on load our list is great rendered to the DOM :clap:  But from the project requirement you need to render them in the order of their index. When I changed the index value the order doesn't change :cry:  Please fix it in order to make your project work as expected :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @AbeeraTahir  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great use of GitHub flow :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Great project implementation :heavy_check_mark: 
- Great UI design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/index.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @AbeeraTahir  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**